sentinel-core/src/main/java/com/alibaba/csp/sentinel/log/CspFormatter.java

```diff
@@ -37,6 +37,7 @@ public String format(LogRecord record) {
         final DateFormat df = dateFormatThreadLocal.get();
         StringBuilder builder = new StringBuilder(1000);
         builder.append(df.format(new Date(record.getMillis()))).append(" ");
+        builder.append(record.getLevel().getName()).append(" ");
```

Comment:
And sentinel-core's logging didn't have level included before? I haven't noticed that.

Reply:
Good, we'd ignored it before. And maybe we can improve the date format in `CspFormatter` to be `yyyy-MM-dd HH:mm:ss.SSS`. It might be helpful to have milliseconds recorded. @cdfive